### DISCUSSION TITLE
Official prettier-plugin-tailwind

### DISCUSSION THREAD

**Discussion by nghiepdev** [tailwindlabs/tailwindcss#6655] [Ideas]
[Prettier Plugin Tailwind](https://github.com/tqwewe/prettier-plugin-tailwind) - Sort tailwind classes in HTML with Prettier.
This is a very useful plugin for Tailwind project. But currently, no longer maintain https://github.com/tqwewe/prettier-plugin-tailwind/issues/46

Hope this project can move to official Tailwind and continue to maintain by the Tailwindlabds team.

**Comment by adamwathan** [comment]
Working on it! We have a prototype already, hoping to polish the final details after Christmas.
